pkg/kubelet/volumemanager/populator/desired_state_of_world_populator_test.go

```diff
@@ -567,6 +567,55 @@ func TestCreateVolumeSpec_Invalid_Block_VolumeMounts(t *testing.T) {
 	}
 }
 
+func TestCreateVolumeSpec_Invalid_Block_VolumeMounts_Disabled(t *testing.T) {
```

Comment:
Oh actually, can you add one more test case for the nil volumeMode?

Reply:
New test added.